zed/CMakeLists.txt

```diff
@@ -0,0 +1,52 @@
+SET(execName ZED_with_OpenCV.so)
+CMAKE_MINIMUM_REQUIRED(VERSION 2.4)
+PROJECT(${execName})
+
+option(LINK_SHARED_ZED "Link with the ZED SDK shared executable" ON)
+
+if (NOT LINK_SHARED_ZED AND MSVC)
+    message(FATAL_ERROR "LINK_SHARED_ZED OFF : ZED SDK static libraries not available on Windows")
+endif()
+
+if(COMMAND cmake_policy)
+	cmake_policy(SET CMP0003 OLD)
+	cmake_policy(SET CMP0015 OLD)
+endif(COMMAND cmake_policy)
+
+SET(EXECUTABLE_OUTPUT_PATH ".")
+
+find_package(ZED 3 REQUIRED)
+find_package(OpenCV REQUIRED)
+find_package(CUDA ${ZED_CUDA_VERSION} EXACT REQUIRED)
```

Comment:
This reminds me: ZED SDK needs to be part of `install_deps.sh` now. But this would be tricky because ZED SDK has very specific requirements for OS and CUDA version. So we'll have to detect which OS and CUDA the user already has. And speaking of CUDA, should we add that as a dependency as well or leave that up to the user? We have so far been assuming that the user already has it.

Reply:
We should leave the ZED and CUDA installation to the user. I will add a README to the ZED plugin on how to do this. We should add a section for ZED in the ILLIXR docs as well.

Reply:
Why leave them up to the user? We install every other dependency for them, so why leave these two?